### DISCUSSION TITLE
Support for Current Linux/ROS Releases

### DISCUSSION THREAD

**Discussion by william-pope** [prl-mushr/mushr#74] [Software]
Not sure if this has been discussed on this forum, but the documentation online only mentions the use of Ubuntu 16.04 and 18.04, and ROS1 Kinetic and Melodic for this project. 

Since these systems are becoming somewhat outdated, I was planning to use more current versions of Linux and ROS in my implementation. How much of the released code would be compatible with Ubuntu 20.04 and ROS1 Noetic? And is ROS2 a possibility now or in the future?

Appreciate anyone's input on this!

**Comment by schmittlema** [comment]
Yes so we have been working hard to migrate to python3/noetic for the past 6 months. Ubuntu 20.04 does not run on Jetsons (at least not officially) and will not be supported. We have created a docker container to run noetic/mushr on both laptops and the car. We are close to releasing it, but are first fixing our mushr_rhc (controller) code to be compatible and creating new documentation/tutorials to run it. If you would like to try it with minimal instructions I can pass along the information now.

**Reply by william-pope** [reply]
Hi Matt, that would be great if you could share it, my email is william.pope@colorado.edu. Thanks!

**Reply by jmcnulty7070** [reply]
Can I receive information on Noetic? Also should Ubuntu be 20.04 focal for Noetic?
